config/routes.rb

```diff
@@ -3,6 +3,8 @@
   root to: "home#index"
   get '/login', to: "sessions#new"
   post '/login', to: "sessions#create"
-  get '/breeds/:name', to: "breeds#show"
+  get '/breeds', to: "breeds#index"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.